### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by afishman**
Updates the readme to contain the latest info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,77 +1,94 @@
...
+- `SeqSero2`
+- `Seqtk`
+- `Shovill`
+- `sistr`
+- `SRST2`
+- `sratoolkit`
+
+# Running the pipeline
+
+To run the pipeline on a batch a samples, the raw `.fastq.gz` files must be stored in `~/WGS_Data/<runID>`.  Each read-pair sample is represented by a pair of files named `*_R1.fastq.gz` and `*_R2.fastq.gz`. For example, to batch two samples named `salmonella_a` and `salmonella_b`, a `~/WGS_Data/<runID>` directory containing four files is required: `salmonella_a_R1.fastq.gz`, `salmonella_a_R2.fastq.gz`,  `salmonella_b_R1.fastq.gz` and `salmonella_b_R2.fastq.gz`, needs to be defined.
```
Comment (ahussaini821) [inline README.md:31]: 'batch a samples' to 'batch of samples'

Comment (ahussaini821) [inline README.md:31]: Not sure if the 'needs to be defined' at the end makes grammatical sense but not 100% sure

File: README.md [hunk 2]
```diff
@@ -1,77 +1,94 @@
...
+```
+
+Pipeline output is stored in  `~/WGS_Results/<runID>/` and contains:
+- TODO
+
+## Run from docker
+
+**Note:** While running from the terminal is the easiest method for developers and data analysts, the pipeline can also be run from docker. This method has the benefit of working across platforms while guaranteeing consistency with automated tests (see below). 
+
+A docker image containing all required dependancies is provided [here](https://hub.docker.com/r/jguzinski/salmonella-seq).
```
Comment (ahussaini821) [inline README.md:45]: 'dependancies' to 'dependencies'

File: README.md [hunk 3]
```diff
@@ -1,77 +1,94 @@
...
+Pipeline output is stored in  `~/WGS_Results/<runID>/` and contains:
+- TODO
+
+## Run from docker
+
+**Note:** While running from the terminal is the easiest method for developers and data analysts, the pipeline can also be run from docker. This method has the benefit of working across platforms while guaranteeing consistency with automated tests (see below). 
+
+A docker image containing all required dependancies is provided [here](https://hub.docker.com/r/jguzinski/salmonella-seq). 
+
+This pull the latest image (if it's not already fetched) from dockerhub and run the container on data
```
Comment (ahussaini821) [inline README.md:47]: 'This pull' to 'This pulls'

'run' to 'runs' (?)

File: README.md [hunk 4]
```diff
@@ -1,77 +1,94 @@
...
+```
+sudo docker run jguzinksi/salmonella-seq:prod
+sudo docker run --rm -it -v /ABS/PATH/TO/READS/:/reads/ -v /ABS/PATH/TO/RESULTS/:/results/ jguzinksi/salmonella-seq:prod /root/nextflow/nextflow SCE3_pipeline_update.nf --runID <runID>
+```
+
+# Pipeline Algorithm 
+
+The pipelines processes data in three stages, as shown below. During the preprocessing stage; low quality bases, adapter sequences are removed from the fastq sample file and then subsampled to a maximum of 3M reads. Following this, the analysis stage runs multiple serotyping tools in parallel.
```
Comment (ahussaini821) [inline README.md:55]: 'pipelines processes' to either 'pipeline processes' or 'pipelines process'

Comment (ahussaini821) [inline README.md:55]: 'low quality bases, adapter sequences' to 'low quality bases and adapter sequences'

File: README.md [hunk 5]
```diff
@@ -1,77 +1,94 @@
...
+The automated tests provided here ensure the software runs as expected. If you make changes to the algorithm, it is **strongly** reccomended that you run these tests to verify the pipeline is behaving as intended. The tests are also automatically run by [TEAMCITY](https://apha.teamcity.com/viewType.html?buildTypeId=NextflowSerotypingPipeline_Pipeline&guest=1) on each pull-request. 
+
+The automated tests assert the correct serovar is assigned to known samples. These are called `inclusivity` tests. To run an inclusivity test, call:
+```
+$ bash -e test_isolates/jobs/inclusivity.bash 0
+```
+
+# Release Process
+
+To release a new version of the software, the `master` branch needs only to be merged into `prod` branch. To perform this merge, a pull-request from the `master` branch into the `prod` branch needs to be made. Approval of pull-requests to `prod` is made by the CODEOWNER (Liljana Petrovska). The CODEOWNER is responsible for ensuring the code conforms to the reliability tests. A positive test result is required for approval.
```
Comment (ahussaini821) [inline README.md:83]: "into 'prod branch'" to "into the 'prod branch'"

File: README.md [hunk 6]
```diff
@@ -1,77 +1,94 @@
...
+
+To release a new version of the software:
+1. A developer makes a pull-request from the `master` to `prod` branch. The CODEOWNER is automatically notified by e-mail.
```
Comment (ahussaini821) [inline README.md:86]: "the `master` to `prod` branch" to "the `master` to the `prod` branch"

File: README.md [hunk 7]
```diff
@@ -1,77 +1,94 @@
...
+1. The CODEOWNER ensures the reliability tests pass on the `master` branch and reviews the code changes. 
+1. The CODEOWNER approves the pull-request if they satisfied, or requests changes.
+1. The dev merges the `master` branch into `prod`
+1. Following approval, the developer tags the current of head of master as the next version. Versions are numbered incrementally with integers, for example `v1`, `v2`, etc. This can be performed by navigating to github master branch and selecting `Create a release`
```
Comment (ahussaini821) [inline README.md:90]: "tags the current of head of master" to "tags the current head of master"

**Review by ahussaini821 — CHANGES_REQUESTED** [review]
Minor grammar changes. Some changes I requested are sort of subjective.
Looks great otherwise.

File: README.md (hunk 1 above)
Comment (afishman) [reply README.md:31]: I agree, removed.

**Review by ahussaini821 — APPROVED** [review]
Looks great!

**Pull request merged**